packages/@coorpacademy-components/src/util/time-format.js

```diff
@@ -1,15 +1,14 @@
 function formatMinutes(minutes) {
-  let timeString = '';
+  const timeArr = [];
   const hours = Math.floor(minutes / 60);
-  let remainingMinutes = minutes % 60;
+  const remainingMinutes = `${minutes % 60}`;
```

Comment:
(would have kept remaning minutes as a number, but this is really an insignificant detail here :sweat_smile:)